src/components/20-molecules/datepicker/index.js

```diff
@@ -445,6 +450,20 @@ class AXADatepicker extends NoShadowDOM {
                       )}
                   </div>
                 </div>
+                <button
+                  class="m-datepicker__button m-datepicker__button-prev"
+                  @click=${() => this.showNextMonth(-1)}
+                  ?disabled=${year === this.getNextYear(-1) && month === 0}
```

Comment:
so you can't cross into a previous year by < button? Is that the UX the designers want?

If so, some semantically named variables above the render HTML would reflect that better, like const atBeginningOfYear = ....

Reply:
You can - but only if there is a previous allowed year.